guides/V1_conversation_main.md

```diff
@@ -13,8 +13,8 @@ Watson Work Services represents a Conversation with this object.
 | property      | type          | description  |
 | ------------- |:------------- |:-----|
 | id          | ID      | The ID of the Conversation. The ID scalar type represents a unique identifier, often used to refetch an object or as key for a cache. The ID type appears in a JSON response as a String; however, it is not intended to be human-readable. When expected as an input type, any string (such as "4") or integer (such as 4) input value will be accepted as an ID.|
-| created     | Date        | Date this space  was created |
-| createdBy   | Person    | Person who created this space |
-| updated     | Date    | Date this space  was updated |
-| updatedBy   | Person | Person who updated this space  |
+| created     | Date        | Date space for this conversation was created |
+| createdBy   | Person    | Person who created space for this conversation |
+| updated     | Date    | Date space for this conversation  was updated |
+| updatedBy   | Person | Person who updated space for this conversation  |
```

Comment:
Minor nitpick but could we just rephrase these to sentences instead of phrases. Had to do a double take before realizing what was meant, which means it may be tough for folks who aren't familiar with the terminology.

Reply:
Yea I agree with Vijay here - eg 

The date this space for this conversation was created.